back/engines/commercial/verification/app/controllers/verification/web_api/v1/verification_methods_controller.rb

```diff
@@ -21,21 +21,21 @@ def index
 
         def first_enabled
           method = VerificationService.new.first_method_enabled
-          authorize method, policy_class: VerificationMethodPolicy if method
```

Comment:
This was causing an internal server error. If method was nullish, it would not call `authorize`, resulting in the 'authorize not called' rails thing